### ISSUE TITLE
OR condition doesn't work along with $exist

### ISSUE THREAD

**Issue by claylua** [balderdashy/sails-mongo#170]
Try firing this on mongodb will yield a results

{
    "$or": [
        {
            "status": {
                "$exists": false
            }
        },
        {
            "status": 1
        }
    ]
}

However, on waterline mongo, the following will not work and only parse numeric value and not objects

Media.find({
status: [{"$exist":false}, 1]
}).done(function(err, obj){

});

it will just read the numeric value and not the object provided.

**Comment by sailsbot** [comment]
Thanks for posting, @claylua.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
